pkg/controllers/user/resourcequota/resource_quota_sync.go

```diff
@@ -48,7 +51,66 @@ func (c *SyncController) syncResourceQuota(key string, ns *corev1.Namespace) (ru
 	}
 
 	_, err := c.CreateResourceQuota(ns)
-	return nil, err
+	if err != nil {
+		return nil, err
+	}
+
+	return nil, c.createLimitRange(ns)
+}
+
+func (c *SyncController) createLimitRange(ns *corev1.Namespace) error {
+	existing, err := c.getExistingLimitRange(ns)
+	if err != nil {
+		return err
+	}
+
+	limitRangeSpec, err := c.getNamespaceLimitRange(ns)
+	if err != nil {
+		return err
+	}
+
+	limitRangeToUpdate, err := c.getResourceLimitToUpdate(ns)
+	if err != nil {
+		return err
+	}
+
+	operation := "none"
+	if existing == nil {
+		if limitRangeSpec != nil {
+			operation = "create"
+		}
+	} else {
+		if limitRangeSpec == nil {
+			operation = "delete"
+		} else if limitRangeToUpdate != "" || limitsChanged(existing.Spec.Limits, limitRangeSpec.Limits) {
```

Comment:
just curious, why don't we check for `limitsChanged` if `limitRangeToUpdate!=""`?

Reply:
limitRangeToUpdate is supposed to be limits defined on the namespace + completed by the limits defined on the project, if some fields are nil. But now I have realized I'm not passing the `limitRangeToUpdate` to the update call. Will work on it